### DISCUSSION TITLE
Sharing folders located outside of Synctrain sandboxed folder?

### DISCUSSION THREAD

**Discussion by vano21** [pixelspark/sushitrain#1] [Q&A]
This is a use case for apps which do not work with external folders such as Obsidian. I appreciate this is not a simple issue to resolve. Are there any known workarounds for being able to share external folders, and, this is a long shot, any plans to implement external folders for Synctrain? 

This iOS implementation works surprisingly well, and dare say even simpler to set up than using Syncthing's web ui. Great work!

**Comment by pixelspark** [comment]
I know MobiusSync supports it, but I haven't looked into how it works regarding permissions on iOS.

If you are adventurous, you can try to see if it works by placing a custom config.xml  (with a folder config path pointing to a folder outside the Synctrain folder) in the Synctrain folder root on iOS and restarting the app (I can provide a default config if needed). I expect that the app would first need to request permission to access the external folder however.

**Comment by pixelspark** [comment]
Note that the trick with custom config.xml is not going to work as the app will rewrite the folder paths each time it reads the configuration. For this feature I'd have to investigate feasibility first.

**Comment by pixelspark** [comment]
As of v1.3, the app wil not rewrite folders paths anymore if the configured path points to a valid folder (having the `.stfolder` marker). 

Still it may not be possible to simply insert a path to another app's container without actually first gaining permission from the user to acces (I would have to check this out).

**Comment by pixelspark** [comment]
This feature is available in the next version that is currently on TestFlight.
